tests/unit/cpp_api.cpp

```diff
@@ -471,3 +471,27 @@ TYPED_TEST(CppApiTestFixture, TestNodeProperties) {
   ASSERT_EQ(node_1.Properties()["b"].ValueString(), "b");
   ASSERT_EQ(node_1.GetProperty("b").ValueString(), "b");
 }
+
+TYPED_TEST(CppApiTestFixture, TestValueOperatorLessThan) {
+  const int64_t int1 = 3;
+  const int64_t int2 = 4;
+  const double double1 = 3.5;
+  const mgp::Value &int_test1 = mgp::Value(int1);
```

Comment:
You don't need & here, as you are doing assignment, and I am not sure to what is that reference binding, probably to temporary variable and that could result in undefined behaviour